src/ElFinder.php

```diff
@@ -356,7 +356,7 @@ public function __construct($opts) {
                             if (is_string($handler) && strpos($handler, '.')) {
                                 list($_domain, $_name, $_method) = array_pad(explode('.', $handler), 3, '');
                                 if (strcasecmp($_domain, 'plugin') === 0) {
-                                    if ($plugin = $this->getPluginInstance($_name, isset($opts['plugin'][$_name])? $opts['plugin'][$_name] : array())
+                                    if ($plugin = $this->getPluginInstance($_name, isset($opts['plugins'][$_name])? $opts['plugins'][$_name] : array())
```

Comment:
Ok, but in cofig.ylm and in documentation we have 'plugin**s**'
https://github.com/helios-ag/FMElfinderBundle/blame/master/Resources/doc/advanced-configuration.md#L93
such small inaccuracy :)